peer-pod-controller/Dockerfile

```diff
@@ -15,13 +16,11 @@ COPY api/ api/
 COPY controllers/ controllers/
 
 # Build
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o manager main.go
+RUN CGO_ENABLED=1 GOOS=linux GOARCH=amd64 go build -a -o manager main.go
```

Comment:
Can we specify build tags for cloud providers here in the same way of other binariers like `cloud-api-adaptor` and `cluster-provisioner`?

Build tags are derived from `BUILTIN_CLOUD_PROVIDERS` in the main Makefile.
https://github.com/confidential-containers/cloud-api-adaptor/blob/900b330e4502ba219abcd900bdf7c541539e2a1c/Makefile#L21-L26

Reply:
here's tags might work I'll have a look